include/zephyr/storage/stream_flash.h

```diff
@@ -65,6 +65,8 @@ struct stream_flash_ctx {
 #ifdef CONFIG_STREAM_FLASH_ERASE
 	off_t last_erased_page_start_offset; /* Last erased offset */
 #endif
+	uint8_t erase_value;
+	uint8_t write_block_size;	/* Offset/size device write alignment */
```

Comment:
I have now updated to Zephyr 3.7 and this is a bug for me.
The block size on the device I'm using, NXP's LPC55S59, is 512 and of course, we have an issue since this was defined as 1 byte (max size is 512).
This needs to be size_t.

Reply:
Hi. I have provided fix here https://github.com/zephyrproject-rtos/zephyr/pull/76053.
Thanks for reporting the issue.
I just want to mention that you are always free to provide a fix that will go under review, so any doubts you can have, regarding technical solution, can be resolved together in the context of your contribution; so next time you find something like this I would like to encourage you to log an issue and, if you have time, follow it with a fix.